cdk/l2-cdk-resources/third-party-integration/src/microsoftTeamsIntegration.ts

```diff
@@ -0,0 +1,37 @@
+// Copyright 2023 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {
+  CfnThirdPartyIntegration,
+  CfnThirdPartyIntegrationPropsType,
+} from '@mongodbatlas-awscdk/third-party-integration';
+import { Construct } from 'constructs';
+import { ThirdPartyIntegrationProps } from './thirdPartyIntegrationBase';
+
+export interface MicrosoftTeamsIntegrationProps extends ThirdPartyIntegrationProps {
+  readonly microsoftTeamsWebhookUrl: string;
+}
+
```

Comment:
[q] Can we return an error if this prop is not set?

Reply:
It's a required field, so user should get a compilation error if they don't set this